python/ray/tune/automl/genetic_searcher.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# author: yiguang.wyg (yiguang.wyg@antfin.com)
+# date: 2018/06/27
+
+import logging
+import numpy as np
+
+from .search_policy import AutoMLSearcher
+
+
+class GeneticSearch(AutoMLSearcher):
+    """Implement the genetic search.
+
+    Keep a collection of top-K parameter permutations as base genes,
```

Comment:
Can you provide more detail in this docstring? It would be good to know things like; how populations are generated; when is crossover going to occur; when mutation is going to occur; etc.

Reply:
Details written in `_next_generation`